ApolloFitness/navigation/MainTabNavigator.js

```diff
@@ -23,14 +24,7 @@ const HomeStack = createStackNavigator(
 HomeStack.navigationOptions = {
   tabBarLabel: 'Home',
   tabBarIcon: ({ focused }) => (
-    <TabBarIcon
-      focused={focused}
-      name={
-        Platform.OS === 'ios'
-          ? `ios-information-circle${focused ? '' : '-outline'}`
-          : 'md-information-circle'
-      }
-    />
+    <TabBarIcon focused={focused} name={Platform.OS === 'ios' ? 'ios-home': 'md-home'}/>
```

Comment:
Do we need to check for iOS here since we're just planning on building to android?

Reply:
we don't need to, technically we're wasting a few bytes to check, however is it good form and makes the program a little more future proof if we ever decide to build iOS later